ReactWindows/ReactNative.Bridge/JavaScriptModuleBase.Generated1.cs

```diff
@@ -1,4 +1,11 @@
-﻿using System.Runtime.CompilerServices;
+﻿
+
+
+
+
+
+
+using System.Runtime.CompilerServices;
```

Comment:
it actually generated a whole new file and appended the '1' to the name for reasons I don't understand. should we even be checking in the file if it's generated?